### PR TITLE
lisk_snapshot.sh - Bugfix

### PR DESCRIPTION
**Pull request by Gr33nDrag0n69**
DAYS_TO_KEEP cleanup command not working as expected

### CODE REVIEW

File: scripts/lisk_snapshot.sh [hunk 1]
```diff
@@ -163,7 +163,7 @@ dropdb --if-exists "$TARGET_DB_NAME" &> /dev/null
 
 echo -e "\n$(now) Deleting snapshots older then $DAYS_TO_KEEP day(s) in $BACKUP_LOCATION"
 mkdir -p "$BACKUP_LOCATION" &> /dev/null
-find "$BACKUP_LOCATION" -name "${SOURCE_DB_NAME}*.gz" -mtime +"$DAYS_TO_KEEP" -exec rm {} \;
+find "$BACKUP_LOCATION" -name "${SOURCE_DB_NAME}*.gz" -mtime +"$(( DAYS_TO_KEEP - 1 ))" -exec rm {} \;
```
Comment (Isabello) [inline scripts/lisk_snapshot.sh:166]: What about the original implementation doesnt work?

Comment (Gr33nDrag0n69) [reply scripts/lisk_snapshot.sh:166]: keep 1 day over

Comment (Gr33nDrag0n69) [reply scripts/lisk_snapshot.sh:166]: days to keep =2 should result in keeping last 48 hours of files

Comment (Gr33nDrag0n69) [reply scripts/lisk_snapshot.sh:166]: not 72 hours

Comment (Isabello) [reply scripts/lisk_snapshot.sh:166]: If i say days to keep 7. Mtime +7 would keep 7 days and only delete things older than that, ie: 8 days.

Comment (Gr33nDrag0n69) [reply scripts/lisk_snapshot.sh:166]: evrything else look like working perfectly in the last 3 days

Comment (Isabello) [reply scripts/lisk_snapshot.sh:166]: ok, i dont have time to test this now but i will do so later this week.

**Pull request merged**